data/error_policies/derive_non_matching_parents.cas

```diff
@@ -0,0 +1,14 @@
+virtual resource foo {
+	fn do_something(domain source, domain target) {
+		allow(source, target, file, read);
+	}
+}
+
+virtual resource bar {
+	fn do_something(domain source) {
+		allow(source, this, file, read);
+	}
+}
+
+@derive([do_something], union)
```

Comment:
I guess this `union` argument will also go away? It can only confuse users.

Reply:
Yep.  Same as above.  I missed an update in the error tests.  Thanks for catching it